### PR TITLE
fix: channel info modal leaves large empty whitespace on About tab

### PR DESCRIPTION
**Pull request by XyneSpaces**
1. Problem Description:
The channel info modal (About tab) renders with a large block of empty whitespace between the "Created By …" line and the pinned "Channel ID" footer. The modal is forced to a fixed 720px height regardless of content, so the short About tab looks broken/empty.

2. If this is a bug fix or an enhancement, how did you identify the issue?:
Reported in-app (screenshot of the xyne-spaces channel info modal). Traced to `apps/dashboard/src/components/Chat/Info/Info.tsx` — the modal root div hardcodes `h-[720px]`. The Dialog wrapper in `SupportScreen.tsx` sets only `max-w-[620px]` and no height, so the whole modal height comes from that class. On the short About tab this leaves a big gap (the footer is pinned via `mt-auto`).

3. Explain the solution implemented in the PR:
Make the modal height content-driven on the About tab (`max-h-[720px]`) while keeping the fixed `h-[720px]` on the tabs that host a scroll viewport. This is important because the Members tab uses a `Virtuoso` virtualized list with `height: 100%`, which requires a definite parent height — a blanket `max-h` would collapse that list to zero. Change is a one-line className -> cn() conditional keyed off `activeTab`.

4. Documentation (link to docs created/updated for this change): N/A

5. DB Alter Details (if any): N/A

6. Data fix Details (if any): N/A

7. Environment variable Changes (if any): N/A

8. Testing (how it was tested; screenshots/links): Verified the About tab now sizes to its content (no empty gap) and that Members/Notifications/Settings/AI Preference tabs retain the 720px height so their scroll viewports (incl. the Virtuoso members list) still work.

9. Services to which this change is to be deployed: dashboard (frontend only)

10. Main PR link (if this PR is raised against a release branch): N/A (raised against main)

### CODE REVIEW

File: apps/dashboard/src/components/Chat/Info/Info.tsx [hunk 1]
```diff
@@ -293,7 +293,13 @@ const Info = ({
   return (
     <div
       ref={popoverContainerRef}
-      className='overflow-clip h-[720px] bg-background flex flex-col'
+      className={cn(
+        'overflow-clip bg-background flex flex-col',
+        // The About tab is short; a fixed height leaves a large empty gap.
+        // Size to content there, and keep the fixed height on tabs that host
+        // a scroll viewport (Members' virtualized list needs a definite height).
+        activeTab === 'about' ? 'max-h-[720px]' : 'h-[720px]',
```
Comment (XyneSpaces) [inline apps/dashboard/src/components/Chat/Info/Info.tsx:301]: **[major] R7 — review-guidelines #204 (tests for new behavior / edge cases)** — This ternary is the whole invariant of the fix: About sizes to content (`max-h-[720px]`) while every other tab keeps the fixed height because Members' Virtuoso list (`style={{ height: '100%' }}` inside `flex-1 min-h-0`) needs a definite parent height. Nothing locks that in — a future refactor of this className (or an inverted conditional) would silently collapse the Members list with no test failing.

Suggested: a small component test asserting the root class per `activeTab` (e.g. `about` → contains `max-h-[720px]`, `members` → `h-[720px]`), or an e2e step asserting the About modal is content-sized while Members keeps the 720px viewport.

Context: #1543, the previous change to this file, also shipped without a test — if the team treats CSS-only fixes as test-exempt, note that here and this drops to a nit.

File: apps/dashboard/src/components/Chat/Info/Info.tsx [hunk 2]
```diff
@@ -293,7 +293,13 @@ const Info = ({
   return (
     <div
       ref={popoverContainerRef}
-      className='overflow-clip h-[720px] bg-background flex flex-col'
+      className={cn(
```
Comment (XyneSpaces) [inline apps/dashboard/src/components/Chat/Info/Info.tsx:296]: **[nit] R8 — magic-value duplication** — 720px now lives in two sibling literals that must stay in sync (a mismatch gives About and Members different caps — visually subtle and easy to miss). Tailwind needs full literal class strings, but you can still name them: `const MODAL_MAX_H = 'max-h-[720px]'` / `const MODAL_FIXED_H = 'h-[720px]'` next to the component — or leave as-is and let the comment above serve as the sync reminder.

**Review by XyneSpaces — COMMENTED** [review]
Automated review by **Xyne Spaces Reviewer** — 1 file, 1 hunk (+7/−1) checked against the team review guidelines.

The change itself is verified correct: `cn` is imported from `utils/classNames`; `activeTab` is typed against the `ChannelTab` union so the `'about'` comparison is compile-safe; and the Members tab really does host a `Virtuoso` list (`height: 100%` inside `flex-1 min-h-0`) that needs a definite parent height — so capping only the About tab is the right call.

Findings: **0 blocking, 1 major, 2 nits** — see the two inline comments and the summary comment for details and the reviewer assignment.

**Comment by XyneSpaces** [comment]
## Xyne Spaces Reviewer — PR #1554

**Verdict:** Changes requested — 1 major (missing regression test for the height behavior), 2 nits. No blocking issues; the fix itself is correct.

**Reviewed:** 1 file, 1 hunk (+7/−1) · **Findings:** 0 blocking, 1 major, 2 nits

### Findings
| Severity | Rule | Location | Issue |
|----------|------|----------|-------|
| major | R7 (guideline 204) | `apps/dashboard/src/components/Chat/Info/Info.tsx:301` | New tab-conditional sizing has no test — a future className refactor could silently break the Members tab's Virtuoso definite-height requirement with nothing failing |
| nit | R8 | `Info.tsx:296` | The 720px value is now duplicated in two sibling literals that must stay in sync |
| nit | R8 (guideline 294) | PR title / branch | No `XYNE-NNNNN` ticket key — nearly every other PR title in this repo carries one (`fix/` branch prefix ✅) |

**Verified from the diff + file:** `cn` is imported (`utils/classNames`), so this compiles; `activeTab` is typed against the `ChannelTab` union (`'about' | 'members' | 'notifications' | 'settings' | 'ai-features'`), so the comparison is compile-safe; the Members tab really does render a `Virtuoso` list at `height: 100%` inside `flex-1 min-h-0`, so keeping the fixed height off About only is the right call. Tall About content still clips at 720px exactly as before the PR — no regression, just not improved either.

**Worth a manual eyeball:** modal height now changes when switching About ↔ other tabs (intended, but confirm it doesn't feel jumpy); About with a very long channel description still clips at 720px (pre-existing).

### Recommended reviewer
**@Dnyaneshwari-04** — authored #1543, the most recent change to this exact file (merged today), and carries zero pending review requests.

| Candidate | Owns (files in this PR) | Default reviewer? | Pending reviews | Tie-breaker |
|-----------|-------------------------|-------------------|-----------------|-------------|
| @Dnyaneshwari-04 | `Info.tsx` — last author (#1543, merged 2026-09-04) | no | 0 | most recent author of this file; active today |
| @harshpreet931 | `Info.tsx` — reviewed #1543 | no | 0 (not a requested reviewer on any of the newest 100 open PRs) | — |
| @samitbarai | `Info.tsx` — reviewed #1543 | no | 0 (no appearances in the newest 100 open PRs) | — |
| Abhishek Verma / Mahek Agarwal / Rajaprabhu Natrajan | none — no recent commits on this file | no | not computable (not resolvable to GitHub logins) | architect-suggested alternates |

No `.github/CODEOWNERS` entry matches `apps/dashboard/**` (it covers only ACL/mutator paths), so ranking is by file ownership + capacity: the repo has hundreds of open PRs but only 8 of the newest 100 carry a pending reviewer request, so capacity is effectively equal — familiarity decides, and @Dnyaneshwari-04 wrote the last change to this file hours ago.

⚠️ This agent has no reviewer-request API available in this session, so the formal "Reviewers" assignment couldn't be set automatically. @Dnyaneshwari-04 — could you take this one? (Anyone with access can also add her under Reviewers on the PR page.)

**Comment by XyneSpaces** [comment]
**Reviewer recommendation (add-on)** — the automated review already posted its findings. Based on current availability and repo activity, the best assignable reviewer for this dashboard change is **@abhishekverma-droid**.

| Candidate | Mapped GitHub login | Availability signal | Notes |
|-----------|---------------------|---------------------|-------|
| Abhishek Verma | @abhishekverma-droid | Active today, no leave/OOO | Most active of the three over the last ~2 weeks |
| Mahek Agarwal | *not resolved* | Status `💬 afk` | Could not map to a GitHub login with repo contributions |
| Rajaprabhu Natrajan | @Rajaprabhu-Natarajan | `🏡 Working remotely`, active | Confirmed contributor; busier with 2 open PRs of his own |

Please add @abhishekverma-droid as a reviewer on the PR page if the automation was unable to request them directly.
